services/core-web/src/components/noticeOfWork/applications/review/ReviewApplicationFeeContent.js

```diff
@@ -0,0 +1,325 @@
+/* eslint-disable */
+import React, { useState, useEffect } from "react";
+import { PropTypes } from "prop-types";
+import { isNil } from "lodash";
+import { Drawer, Button, Table, Alert } from "antd";
+import RenderField from "@/components/common/RenderField";
+import RenderDate from "@/components/common/RenderDate";
+import { Field, Fields } from "redux-form";
+import { CloseOutlined } from "@ant-design/icons";
+import RenderFieldWithDropdown from "@/components/common/RenderFieldWithDropdown";
+import { numberWithUnitCode } from "@common/utils/Validate";
+import { getDurationText } from "@common/utils/helpers";
+import LinkButton from "@/components/common/LinkButton";
+import { CoreTooltip } from "@/components/common/CoreTooltip";
+import moment from "moment";
+
+const propTypes = {
+  isViewMode: PropTypes.bool.isRequired,
+};
+
+const tableOneColumns = [
+  {
+    title: "Tonnes proposed to be moved in highest producing year",
+    children: [
+      {
+        title: "Placer mine proposed to operate for 5 years or less",
+        dataIndex: "less_than_5",
+        key: "less_than_5",
+      },
+      {
+        title: "Placer mine proposed to operate for more than 5 years",
+        dataIndex: "more_than_5",
+        key: "more_than_5",
+      },
+    ],
+  },
+  {
+    title: "Permit Fee",
+    dataIndex: "permit_fee",
+    key: "permit_fee",
+    width: 100,
+    className: "right",
+  },
+];
+
+const tableTwoColumns = [
+  {
+    title: "Tonnes proposed to be extracted in highest producing year",
+    dataIndex: "tonnes_per_year",
+    key: "tonnes_per_year",
+  },
+  {
+    title: "Permit Fee",
+    dataIndex: "permit_fee",
+    key: "permit_fee",
+    width: 100,
+    className: "right",
+  },
+];
+
+const tableOneData = [
+  { more_than_5: "< 10 000", less_than_5: "< 60 000", permit_fee: "$0" },
```

Comment:
Should we format these amounts with ",". E.g., "10 000" --> "10,000"?

Reply:
Yeah, I can update that